### ISSUE TITLE
Temp_downloads handling

### ISSUE THREAD

**Issue by POE-Addon-Launcher** [POE-Addon-Launcher/PAL2#37]
Add a check that ensures the temp_download folder is an empty folder when setting it.

**Comment by Dissolator** [comment]
It's better to use temp_downloads path as parent for your app's temporary folder, which you can create, fill with files and delete without any risk to harm other users files. Also you can name it with timestamp to minimize risk of coincidence with already existing folders (like "pal"). Something like %temp_downloads%\pal2_05_06_2019\ should be enough

**Comment by POE-Addon-Launcher** [comment]
Yeah, that's also what I was thinking off, I was also thinking of when a user specifies a directory and it isn't empty I just create another folder inside of to use for PAL
